tests/x11/kontact.pm

```diff
@@ -35,7 +35,14 @@ sub run {
         # KF5-based account assistant ignores alt-f4
         wait_screen_change { send_key 'alt-c' } if match_has_tag('test-kontact-1');
     } until (match_has_tag('kontact-window'));
-    assert_and_click 'close_kontact';
+    assert_screen [qw(kontact-error close_kontact)];
+    # workaround for fatal error bsc#1105207
+    if (match_has_tag('kontact-error')) {
+        record_soft_failure('encounter fatal error, bsc#1105207') && return;
```

Comment:
Please use `return record_soft_failure` as we also use that in other places. And also I recommend to phrase as "Encountered fatal error, boo#1105207" to 1) not mistake it as imperative tense and second reference "bugzilla.opensuse.org" which is more appropriate than "bugzilla.suse.com". Afterwards can be merged

Reply:
updated now, thanks